mm2src/coins/utxo.rs

```diff
@@ -1517,7 +1517,7 @@ fn spawn_server_version_retry_loop(weak_client: Weak<ElectrumClientImpl>, client
     }
 
     spawn(async move {
-        while let Some(c) = weak_client.upgrade() {
+        if let Some(c) = weak_client.upgrade() {
```

Comment:
Could you please rename `spawn_server_version_retry_loop ` to something like `check_electrum_server_version`?
This function is not a loop more. And I think we can move `spawn` to `spawn_electrum_version_loop`.